### ISSUE TITLE
Strange behavior for reactors count vs fetched

### ISSUE THREAD

**Issue by netquik** [kevinzg/facebook-scraper#527]
Hi. I'm successfully using the scraper to get reactors from a post.
Anyway today i found a strange behavior that prevent me from fetching the complete list of reactors.

I'm using 

post = next(get_posts(post_urls=[FB_ID], options={"allow_extra_requests": False, "reactions": True, "comments": "generator", "reactors": "generator", "comment_reactors": False}))

i get
Fetching 3000 reactors
Found 50 reactors

but

 'reaction_count': 64,
 'reactions': {'like': 64}

is there a limit for 50 reactors in fetching?

Thanks

**Comment by netquik** [comment]
I did a test i removed one like by my account and it fetches always 50 reactors (adding one that previously was not considered). The problem is that they are not time ordered so can't find a way to keep update also a cache. Why is there this 50 limit i can't see in the extractor code? (also i tried with custom limit of 100. I get always 50 reactors)

**Comment by netquik** [comment]
also note from 
```
set_cookies("cookies.json")
post = next(get_posts(post_urls=["2080102302146892"], options={"allow_extra_requests": False, "reactions": True, "comments": "generator", "reactors": 100, "comment_reactors": False}))
```

it output for `print(len(post['reactors']))` 1440

**Comment by netquik** [comment]
running your code i get:

`2080102302146892 3100 {'like': 2000, 'love': 1000, 'care': 56, 'wow': 17, 'haha': 2, 'angry': 2, 'sad': 1} 3171`

**Comment by neon-ninja** [comment]
Yes, you can see this rounding yourself if you look at https://m.facebook.com/ufi/reaction/profile/browser/?ft_ent_identifier=2080102302146892

**Comment by netquik** [comment]
ok ti works!!!
158876669738713 64 {'like': 64} 64

just a question how can i update to this commit easily? pip does not want to update the code from git (i patched it manually) thanks

**Comment by netquik** [comment]
uninstalling do the job

**Comment by netquik** [comment]
now on docker is a new battle... i forgot how i did... anyway thanks a lot. You are great!
